### ISSUE TITLE
Can you use multiple GPUs?

### ISSUE THREAD

**Issue by KevinLucidyne** [alexgkendall/caffe-segnet#50]
Looking through the code, I'm mainly seeing only a single GPU being used.  Is it possible to use multiple GPUs?

**Comment by jorgyz** [comment]
yes, form the command line, you would do something like this caffe train -solver solver.prototxt -gpu 0,1 where 0 and 1 are the IDs of the GPUs you want to use.  You can also just replace 0,1 with the word 'all' and Caffe will make use of all of the GPUs in your system.
